frontend/schemaCMS/src/routes/page/editPage/editPage.component.js

```diff
@@ -254,6 +275,36 @@ export const EditPage = ({
           </NextButton>
         </ModalActions>
       </Modal>
+      <Modal
+        ariaHideApp={false}
+        isOpen={leavingPageModalOpen}
+        contentLabel={messages.confirmLeavingPage}
```

Comment:
if you use like that, we won't be able to translate it later if needed. You need to use `useIntl` API

Reply:
fixed